### ISSUE TITLE
[Pricing] Make Tier Comparison Header Sticky

### ISSUE THREAD

**Issue by leecalcote** [layer5io/layer5#6122]
### Current Behavior
Site visitors that want to compare features offered between subscription tiers, visit the "Compare All Features" section of the https://layer5.io/pricing page.

### Desired Behavior
While examining the checkmarks and x's in the Compare All Features section, when users scroll down the page, past the names of the subscription tiers and have a hard time recalling which checkmark and which "x" are associated with which subscription tier.

### Screenshots / Mockups
<img width="1256" alt="Screenshot 2024-12-12 at 3 39 01 PM" src="https://github.com/user-attachments/assets/77bb665f-5d68-44a5-a387-1aa1b03dcfea" />

### Implementation
Make the subscription tier table header sticky under the "Compare All Features" section.

### Acceptance Tests
Users can see the subscription tier names at the top of the page.

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by Ajay-singh1** [comment]
@leecalcote Can I work on this issue?

**Comment by leecalcote** [comment]
Sounds great. , @Ajay-singh1

**Comment by eyeaadil** [comment]
@leecalcote , can you assign this issue to me?

**Comment by leecalcote** [comment]
@Ajay-singh1 how are things progressing here?

**Comment by krishkumar84** [comment]
hey @leecalcote kindly assign this issue to me in past i also contribute to this repo i also have local setup also of this repo also

**Comment by Ajay-singh1** [comment]
@eyeaadil Would you like to work on this issue?Also can you confirm if you have the setup the local development environment?

**Comment by eyeaadil** [comment]
> @eyeaadil Would you like to work on this issue?Also can you confirm if you have  setup the local development environment?

Yes, i would like to work in this issue and I setup the local development environment.

**Comment by Ajay-singh1** [comment]
Okay @eyeaadil!

**Comment by leecalcote** [comment]
@Ajay-singh1, you’ve tossed in the towel on your attempt to complete this, correct?

@krishkumar84, how are you progressing?

@eyeaadil, I recall you asking to be assigned too many, many different issues. I don’t recall any of your PR’s being successful. More self-study needed before volunteering again.

**Comment by krishkumar84** [comment]
hey @leecalcote i raised the pr for this issue couple of days ago #6138 kindly review this

**Comment by Ajay-singh1** [comment]
@leecalcote No! Never.I will take this up anytime.Waiting for the Gatsby fix!!

**Comment by Rajesh-Nagarajan-11** [comment]
@vr-varad can you assign this issue to me ?

**Comment by vr-varad** [comment]
Thanks @Rajesh-Nagarajan-11 for taking up this issue.

**Comment by priyanshu4999** [comment]
@vr-varad @Rajesh-Nagarajan-11 can i contribute to this issue, i have seen something similar, in using table for tabulating, it equates to compromise between scroll and sticky headers. If sticky header is required then horizonatal scrolling on thinner width will be compromised. Only working solution i found was tabulating using divs , compromisisng seo.
Could I be assigned this issue.?

**Comment by Rajesh-Nagarajan-11** [comment]
@priyanshu4999 If you’ve set this up locally, please go ahead.

**Comment by Rajesh-Nagarajan-11** [comment]
@priyanshu4999  We can't compromise Horizontal scrolling as well , it needed for mobile view

**Comment by priyanshu4999** [comment]
@Rajesh-Nagarajan-11 , Yes thats what i thought, then recreating tabulation with divs was the solution that worked for both scrolling and sticky, i think there is a whole disscussion regarding same on CSS-tricks. Do tell if you need help or come up with different solution.!!

**Comment by Rajesh-Nagarajan-11** [comment]
@priyanshu4999 You can take over this issue and work it on , @vr-varad  Unassign me and assign this to @priyanshu4999 !

**Comment by vr-varad** [comment]
@priyanshu4999 Its your's.

**Comment by rishiraj38** [comment]
#7665
